### PR TITLE
feat: [account] commission rate

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

Closes: #XXXX [BDU-433](https://forbole.atlassian.net/browse/BDU-433)

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] ran linting
- [ ] wrote tests where necessary
- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] targeted the correct branch
- [x] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] added an entry to the `CHANGELOG.md` file

### CODE REVIEW

File: packages/ui/src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx [hunk 1]
```diff
@@ -28,6 +29,7 @@ const Desktop: React.FC<{
         />
       ),
       amount: `${amount} ${x.amount?.displayDenom.toUpperCase()}`,
+      commission: `${commission} %`,
```
Comment (teamchong) [inline packages/ui/src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx:32]:
```suggestion
      commission: numeral(x.commission).format('0%'),
```
suggest to format using `numeral` and remove `const commission = x.commission ? x.commission : '';`

File: packages/ui/src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx [hunk 2]
```diff
@@ -41,6 +41,14 @@ const Mobile: React.FC<{
                   {x.amount?.displayDenom.toUpperCase()}
                 </Typography>
               </div>
+              <div className={classes.item}>
+                <Typography variant="h4" className="label">
+                  {t('commission')}
+                </Typography>
+                <Typography variant="body1" className="value">
+                  {x.commission ? `${x.commission} %` : ''}
```
Comment (teamchong) [inline packages/ui/src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx:49]:
```suggestion
                  {numeral(x.commission).format('0%')}
```

Comment (rachelhox) [reply packages/ui/src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx:49]: @teamchong wouldn't this `numeral` function converts the `(commission * 100) %` ? As my understanding is that the `commission` from hooks is already `*100 `: https://github.com/forbole/big-dipper-2.0-cosmos/blob/85bed3e38a198f5e52a6f99a7689de5d01fc2e5d/packages/ui/src/screens/account_details/components/staking/hooks.ts#L78)

http://numeraljs.com/#format

Comment (teamchong) [reply packages/ui/src/screens/account_details/components/staking/components/delegations/components/mobile/index.tsx:49]: @rachelhox yes, you are right

**Review by teamchong — APPROVED** [review]

**Pull request merged**